profiles/opentelemetry_dev/compose.yaml

```diff
@@ -0,0 +1,65 @@
+---
+version: "3.7"
+
+services:
+  _base:
```

Comment:
The compose.yaml file should only contain the changes that are needed for your profile. Don't copy the whole compose.yaml file from the base profile. When the system starts up it merges all the compose files, so you just have to declare the services, volumes, networks etc that you need for this profile.